webviz_subsurface/plugins/_co2_leakage/_utilities/generic.py

```diff
@@ -0,0 +1,35 @@
+import pathlib
+from enum import Enum
+from typing import Dict, Iterable, Optional
+
+import numpy as np
+from fmu.ensemble import ScratchEnsemble
+
+
+def fmu_realization_paths(ensemble_path) -> Dict[int, str]:
```

Comment:
Any function that works directly with the filesystem must be wrapped with the `webvizstore` decorator or use a provider approach. This is because if the webviz application is made "portable" and moved e.g. to radix, there will no longer be access to the original files.

How to solve it is a longer discussion which we can have in a meeting :)

Reply:
Removed all direct usage of the two FMU path-related functions in this file. The functionality was easily replaced by using ´read_csv´ from `webvizstore_functions`. I believe this resolves the issue properly.

Well picks and the boundary polygons are now dealt with similarly to how well picks are handled in MapViewerFMU. I guess the main drawback is that this does not allow for differing well picks or boundary polygons across realizations:
- For well picks, I don't know if this is a concern? If it is, then it should also be solved for MapViewerFMU?
- For the boundary polygon, this may be an issue in case the boundary polygon is derived from other data sources that differ between realizations. E.g. if it is derived from the structural model (faults, horizons, etc.). That being said, at this stage it is not critical to fix.